src/io/initrd.rs

```diff
@@ -39,10 +40,14 @@ pub fn make_initrd(members: &[(&str, &[u8])]) -> Result<Vec<u8>> {
 }
 
 /// Extract a compressed or uncompressed CPIO archive and return the
-/// contents of the specified path.
-pub fn extract_initrd<R: Read>(reader: R, path: &str) -> Result<Option<Vec<u8>>> {
+/// contents of paths matching the specified glob patterns.
```

Comment:
Since we're filtering out anything that isn't a regular file, should this say e.g. "files with paths matching ..."?